components/brave_rewards/resources/rewards_page/components/home/benefits_card.style.ts

```diff
@@ -29,6 +29,7 @@ import { scoped } from '../../lib/scoped_css'
       --leo-icon-size: 28px;
       --leo-icon-color: ${color.white};
 
+      min-width: 48px;
```

Comment:
Without this, the icon was being stretched vertically when the text expanded beyond a single line.

Reply:
Thanks for catching this - I think this is flexbox shrinking the element. Doing:

```css
flex-shrink: 0;
```

or

```css
flex: 0 0 auto;
```

might also work.

Reply:
Nice, I went with `flex-shrink` and that seems to work great. Thanks!